Specs/DateTime/English/DateTimeModelTasksMode.json

```diff
@@ -88,5 +88,322 @@
     },
     "NotSupported": "javascript, python, java",
     "Results": []
+  },
+  {
+    "Input": "schedule me a 1:1 meeting",
+    "Comment": "Tasksmode supress 1:1 from getting categorised as time.",
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T00:00:00"
+    },
+    "NotSupported": "javascript, python, java",
+    "Results": []
+  },
+  {
+    "Input": "schedule me a 1:1 meeting on 14 nov",
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T00:00:00"
+    },
+    "NotSupported": "javascript, python, java",
+    "Results": [
+      {
+        "Text": "14 nov",
+        "Start": 29,
+        "End": 34,
+        "TypeName": "datetimeV2.date",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "XXXX-11-14",
+              "type": "date",
+              "value": "2015-11-14"
+            },
+            {
+              "timex": "XXXX-11-14",
+              "type": "date",
+              "value": "2016-11-14"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "schedule me a 1:1 on saturday",
+    "Context": {
+      "ReferenceDateTime": "2022-04-27T00:00:00"
+    },
+    "NotSupported": "javascript, python, java",
+    "Results": [
+      {
+        "Text": "saturday",
+        "Start": 21,
+        "End": 28,
+        "TypeName": "datetimeV2.date",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "XXXX-WXX-6",
+              "type": "date",
+              "value": "2022-04-23"
+            },
+            {
+              "timex": "XXXX-WXX-6",
+              "type": "date",
+              "value": "2022-04-30"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "schedule me a 1:1 meeting 14 nov 2023",
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T00:00:00"
+    },
+    "NotSupported": "javascript, python, java",
+    "Results": [
+      {
+        "Text": "14 nov 2023",
+        "Start": 26,
+        "End": 36,
+        "TypeName": "datetimeV2.date",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "2023-11-14",
+              "type": "date",
+              "value": "2023-11-14"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "Listen to 1990s music",
+    "Comment": "Taskmode will supress number with s (90s) from getting be categorised as daterange",
+    "Context": {
+      "ReferenceDateTime": "2018-05-29T00:00:00"
+    },
+    "NotSupported": "javascript, python, java",
+    "Results": []
+  },
+  {
+    "Input": "Who are us presidents in 1990 s.",
```

Comment:
Should there be the whitespace before the 's'? 
Also, ideally there should be a 1-1 map of specs in the new mode and their versions in the default mode. Is this the only missing? Why?

Reply:
>Since the following example, "Who are us presidents in the 1990s," already existed in default mode, I simply copied it and modified the outcome under tasksmode.